devlib/target.py

```diff
@@ -1035,6 +1035,18 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def adb_reboot_bootloader(self):
+        adb_command(self.adb_name, 'reboot-bootloader', timeout=30)
```

Comment:
That's quite like... but if we keep:

```
def adb_reboot_bootloader(self, timeout=30):
```

that should five more flexibility in case that's required.

Or do you prefer to avoid exposing potential "platform dependencies"?